tests/primer/test_primer_stdlib.py

```diff
@@ -39,14 +39,15 @@ def _patch_stdout(out):
 MODULES_NAMES = [m[1] for m in MODULES_TO_CHECK]
 
 
-@pytest.mark.primer_stdlib
+@pytest.mark.primer
```

Comment:
I think we might want to keep these marks separate?
Then we can create a new `workflow` file with 3 jobs:
1) Prime stdlib
2) Prime batch 1 of external
2) Prime batch 2 of external

Reply:
The issue I encountered that I was not able to take the proper value for adopt in the tox configuration.  https://github.com/PyCQA/pylint/pull/5173#issuecomment-974858646 so the acceptance tests were launched in out standard job taking more time then necessary

Reply:
Does this work:
https://stackoverflow.com/questions/60761243/how-to-specify-several-marks-for-the-pytest-command

Using `and`.

Reply:
Your google-fu is strong.

Reply:
ad611a5